docs/dao/index.md

```diff
@@ -35,6 +35,7 @@ This section functions as the single source of truth of JAM Implementer DAO Memb
 | JAMdotTech \| PyJAMaz | 15gPiSBxhrrQFfShFbrnsQK7kokgQtVs3SEh38YjCDC31de3 | [Link](https://polkadot.subscan.io/extrinsic/25331656-2) |
 | MORUM | 123KEnwuHKiu48WsfwAX4YonGFTpkEmWS6V2WGoJA1qzBkbS | [Link](https://polkadot.subscan.io/extrinsic/25330323-2) |
 | Tessera | 15p3jWZaP4dHTkDTuKM5VXQL5XGfH9U5r6Ntu3UAv2K7vPb8 | [Link](https://polkadot.subscan.io/extrinsic/25331398-2) |
+| Fluffy Labs \| typeberry | 15fGrDWmFoaApnZXEw1Zg45zuCqpkbMS9YPzczeU9dHseUUu | [Link](https://polkadot.subscan.io/multisig_extrinsic/25345876-2) |
```

Comment:
I did not even know this `/multisig_extrinsic/`, nice.

Reply:
@ggwpez @tomusdrw The issue with having multisigs for the team address is that it turns out that the Subsquare (under setup) and maybe Discord requires just ordinary signing of a message rather than an on-chain action.  

So for both Fluffy Labs + JAM Brains use of multisig this may pose a voting mechanics problem.  Allow me to confirm.

Reply:
AFAIK the [Chaos DAO discord bot](https://github.com/ChaosDAO-org/InternalGov-2.0) does not do any verification at all (i.e. you just vote within discord with your discord identity), not sure if that's the one you were planning to use. Don't know much about subsquare, but on Polkassembly we were not able to use or even link our multisig with their internal account.

@ggwpez indeed subscan seems to have some random issues.

If it's preferred for us to just use a regular account let me know. I didn't know it's intended to use that account for the actual voting, and if yes, then it's going to be pretty cumbersome for us as well and I'd rather change it to a different, dedicated account.

Reply:
Would also be good to know if Multisig accounts are valid, since our team is using one as well.

But we could still merge this and accept it as valid application and then lateron figure out if we need different accounts.

Reply:
For Subsquare support which has now been provisioned at , addresses should not be:
* multisig addresses
* pure proxy address or 
* any address that can not sign msg directly
so yes, for both Fluffy + JAM Brains lets go there.  If one of you can amend the membership requirement with this tech detail that would help the next team.